src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
 // write code here
+const wallElement = document.querySelector('.wall');
+const spiderElement = document.querySelector('.spider');
+
+const currentSpiderWidth = spiderElement.clientWidth / 2;
+const currentSpiderHeight = spiderElement.clientHeight / 2;
+const spiderPositionVertical = wallElement.clientWidth / 2 - currentSpiderWidth;
+const spiderPositionHorizontal =
+  wallElement.clientHeight / 2 - currentSpiderHeight;
```

Comment:
Similarly, the variable `spiderPositionHorizontal` is calculating the vertical position. Consider renaming it to `spiderPositionVertical` for consistency and clarity.